src/Raven.Server/Config/Categories/LogsConfiguration.cs

```diff
@@ -69,21 +69,21 @@ public sealed class LogsConfiguration : ConfigurationCategory
 
         [Description("Location of NuGet packages cache")]
         [DefaultValue("Packages/NuGet/Logging")]
-        [ConfigurationEntry("Logs.NuGetPackagesPath", ConfigurationEntryScope.ServerWideOnly)]
+        [ConfigurationEntry("Logs.NuGet.PackagesPath", ConfigurationEntryScope.ServerWideOnly)]
         public PathSetting NuGetPackagesPath { get; set; }
 
         [Description("Default NuGet source URL")]
         [DefaultValue("https://api.nuget.org/v3/index.json")]
-        [ConfigurationEntry("Logs.NuGetPackageSourceUrl", ConfigurationEntryScope.ServerWideOnly)]
+        [ConfigurationEntry("Logs.NuGet.PackageSourceUrl", ConfigurationEntryScope.ServerWideOnly)]
```

Comment:
Package vs Packages. How about we also do a dot based approach? Logs.Packages.X?

Reply:
or Logs.NugetPackages....?